src/bun.js/bindings/webcore/WebSocket.cpp

```diff
@@ -1158,15 +1143,19 @@ void WebSocket::didReceiveBinaryData(const AtomString& eventName, const std::spa
     // });
 }
 
-void WebSocket::didReceiveClose(CleanStatus wasClean, unsigned short code, WTF::String reason)
+void WebSocket::didReceiveClose(CleanStatus wasClean, unsigned short code, WTF::String reason, bool isConnectionError)
 {
     // LOG(Network, "WebSocket %p didReceiveErrorMessage()", this);
     // queueTaskKeepingObjectAlive(*this, TaskSource::WebSocket, [this, reason = WTFMove(reason)] {
     if (m_state == CLOSED)
         return;
+    const bool wasConnecting = m_state == CONNECTING;
```

Comment:
can't this be `isConnectionError` when the function is called?
```
 didReceiveClose(CleanStatus::NotClean, 1006, message, m_state == CONNECTING);
 ```

Reply:
no, see the next line